packages/faq-extension/faq.md

```diff
@@ -29,18 +41,21 @@ Notebook. Here are a few of them you may want to try out:
 
 * Arrange multiple notebooks, terminals, text files, etc. in the
   application using drag and drop.
-* Run code interactively outside of a notebook in the Code Console,
-  and connect one  to a text file.
-* Right click on a markdown file and "Open with..." a live markdown
-  viewer.
+* Run code interactively outside of a notebook in the Code Console, and
+  connect one to a text file.
+* Right click on a markdown file and "Open with..." a live markdown viewer.
 * Double click on CSV files to view them as a nicely formatted table.
 * Drag and drop notebook cells within a notebook or between notebooks.
 
+See the [JupyterLab Documentation](http://jupyterlab.readthedocs.io/en/latest/)
+for more detailed information about these and other features.
+
 ### How stable is JupyterLab?
 
-This beta version of JupyterLab will be characterized by:
+This Beta version of JupyterLab is ready for you to use! More specifically,
+it is characterized by:
 
-* Stable and featureful enough for daily usage by most Jupyter users.
+* Stable and featureful enough for daily usage by users.
```

Comment:
I think you could end this bullet with “daily usage”

Reply:
fixed

Reply:
I guess we don't have to address how stable it is for non-users :-)

Reply:
It’s super stable for non-users! They shouldn’t have experience any problems!